sailfish-k8s/sailfish/overlay/runner-job.yaml

```diff
@@ -24,16 +24,20 @@
             secretKeyRef:
               name: sailfish-broker-credentials-secret
               key: AMQ_PASSWORD
+        - name: MY_POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
         - name: HOST
-          value: sailfish-broker-hdls-svc.rh-summit-demo.svc.cluster.local    # overwritten in overlay
+          value: sailfish-broker-hdls-svc.$(MY_POD_NAMESPACE).svc.cluster.local
         - name: QUEUE_PORT
           value: '5672'
       resources:
         requests:
-          memory: "2000M"
-          cpu: "1000m"
+          memory: 2G
```

Comment:
we should make these consistent in the project. I suggest this one to be the right annotations